auth/src/main/java/com/firebase/ui/auth/provider/FacebookProvider.java

```diff
@@ -43,7 +43,10 @@
 import java.util.List;
 
 public class FacebookProvider implements IdpProvider, FacebookCallback<LoginResult> {
+    public static final String PUBLISH_PERMISSION = "publish";
```

Comment:
I don't think this should be here because it suddenly makes `FacebookProvider` public api, but I'm not sure where to put it that makes sense. Would `AuthUI.FACEBOOK_PUBLISH_PERMISSION` work? Or should we not even make it a constant?